src/crypto_helper/info.rs

```diff
@@ -31,6 +31,11 @@ fn get_algorithm_info(algorithm: &Algorithm) -> Html {
             <a href={"https://www.rfc-editor.org/rfc/rfc4634"}>{"RFC"}</a>{"."}
             </span>
         },
+        Algorithm::Sha384(_) => html! {
+            <span>{"Hash hex-encoded input using SHA384 hashing function."}
```

Comment:
now for md5/sha-XXX algos the user can enter not only hex-encoded input. can you change this and other messages to smth like "Input for XXXX hashing function"?  (you can use a better message if you have such one)